.drone.yml

```diff
@@ -9,13 +9,79 @@ pipeline:
     commands:
       - make build
 
+  postgres:
+    image: postgres:9.6.15
+    name: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    detach: true
+    when:
+      event: push
+
+  prisma:
+    image: prismagraphql/prisma:1.34.8
+    name: prisma
+    environment:
+      PRISMA_CONFIG: |
+        port: 4466
+        databases:
+          default:
+            connector: postgres
+            uri: "postgres://postgres:postgres@postgres:5432/postgres"
+            migrations: true
+            rawAccess: true
+    detach: true
+    when:
+      event: push
+
+  # start houston
+  houston:
+    image: astronomerinc/ap-houston-api:dev
+    name: houston
+    environment:
+      NODE_ENV: "development"
+      WEBSERVER__PORT: "8871"
+      PRISMA__ENDPOINT: "http://prisma:4466/houston"
+      PRISMA__DEBUG: "false"
+      ORBIT__PORT: "8080"
+      DATABASE__CONNECTION: "postgres://postgres:postgres@postgres:5432/postgres"
+      PUBLIC_SIGNUPS: "true"
+      EMAIL_CONFIRMATION: "false"
+      EMAIL__ENABLED: "false"
+      JWT__PASSPHRASE: "secretpassword"
+      DEPLOYMENTS__DATABASE__CONNECTION: "postgres://postgres:postgres@postgres:5432/postgres"
+      DEPLOYMENTS__DATABASE__ENABLED: "true"
+      DEPLOYMENTS__LOG_HELM_VALUES: "true"
+      HELM__RELEASE_VERSION: "0.7.5"
```

Comment:
it's copy/paste from houston drone.yml config. @schnie @samblackk can i remove it?

Reply:
Not sure if it can be removed, but it should only matter for stamping out a new Airflow deployment. This determines the version its going to use. If we are not actually `helm install`ing, the it probably doesn't matter.